### PR TITLE
Tooltip is not updated immediately when a property receives Warning after Error

### PR DESCRIPTION
**Pull request by Sainyam2003**
Fix for #250

### CODE REVIEW

File: Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs [hunk 1]
```diff
@@ -36,5 +36,7 @@ public interface IPropertyGridControlFactory
         /// <param name="tab">The tab.</param>
         /// <returns></returns>
         ContentControl CreateErrorControl(PropertyItem pi, object instance, Tab tab, PropertyControlFactoryOptions options);
+
```
Comment (objorke) [inline Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs:39]: missing comment/explanation

File: Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs [hunk 2]
```diff
@@ -36,5 +36,7 @@ public interface IPropertyGridControlFactory
...
+        FrameworkElement SetValidationErrorStyle(FrameworkElement control, PropertyControlFactoryOptions options);
```
Comment (objorke) [inline Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs:40]: do we need the return value? we have not used "fluent" syntax in this library

File: Source/PropertyTools.Wpf/PropertyGrid/PropertyControlFactoryOptions.cs [hunk 3]
```diff
@@ -29,5 +30,7 @@ public class PropertyControlFactoryOptions
         /// The validation error template.
         /// </value>
         public DataTemplate ValidationErrorTemplate { get; set; }
+
```
Comment (objorke) [inline Source/PropertyTools.Wpf/PropertyGrid/PropertyControlFactoryOptions.cs:33]: missing comment

File: Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs (hunk 2 above)
Comment (Sainyam2003) [reply Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs:40]: did not understand this, this is an interface and we can not return anything from here

Comment (objorke) [reply Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs:40]: the method interface defines a `FrameworkElement` to be returned, could it be `void`?

Comment (Sainyam2003) [reply Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs]: fixed.

**Pull request merged**